packages-ts/gauntlet-terra-contracts/src/commands/contracts/link/deploy.ts

```diff
@@ -1,9 +1,13 @@
 import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
 import { Result } from '@chainlink/gauntlet-core'
 import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
-import { CONTRACT_LIST } from '../../../lib/contracts'
 import { CATEGORIES } from '../../../lib/constants'
 
+const CW20_BASE_CODE_IDs = {
```

Comment:
Why are these hardcoded here instead declared at `codeIDs/{network}.json` file?

Seems inconsistent with:

- packages-ts/gauntlet-terra-contracts/codeIds/bombay-testnet.json
- packages-ts/gauntlet-terra-contracts/codeIds/mainnet.json

Reply:
Those are constants for the cw20 token: https://docs.terraswap.io/docs/contract_resources/contract_addresses/
They should probably moved somewhere else

Reply:
GH Issue:

- https://github.com/smartcontractkit/chainlink-terra/issues/28